src/services/Content/ContentApi.ts

```diff
@@ -30,21 +40,42 @@ export interface ContentListResponse {
   meta: Meta;
 }
 
-export interface ContentRepoParamsResponse {
-  distribution_versions: Array<{ name: string; label: string }>;
-  distribution_arches: Array<{ name: string; label: string }>;
+export interface RepositoryParamsResponse {
+  distribution_versions: Array<NameLabel>;
+  distribution_arches: Array<NameLabel>;
 }
 
-export type filterData = {
+export type NameLabel = {
+  name: string;
+  label: string;
+};
+
+export type FilterData = {
   searchQuery: string;
   versions: Array<string>;
   arches: Array<string>;
 };
 
+export type ValidateItem = {
```

Comment:
what's the "Item" here in ValidateItem?

Reply:
I suppose it would be "object" or "type", not fussy, can change the name. Thoughts? or perhaps "ValidationItem"?

Reply:
Item is okay. I understand what it is now.